packages/eslint/index.js

```diff
@@ -1,46 +1,15 @@
-const Future = require('fluture');
 const deepmerge = require('deepmerge');
-const clone = require('lodash.clonedeep');
-const {
-  assoc, curry, evolve, keys, omit, pathOr, pipe, prop, reduce
-} = require('ramda');
-const { basename, join } = require('path');
+const { join } = require('path');
 
 const MODULES = join(__dirname, 'node_modules');
-const getEslintOptions = config => config.module.rule('lint').use('eslint').get('options');
-const renameKeys = curry((definition, obj) =>
-  reduce((acc, key) => assoc(definition[key] || key, obj[key], acc), {}, keys(obj)));
-const reduceToTrueKeys = reduce((acc, key) => assoc(key, true, acc), {});
-const getEslintRcConfig = pipe(
-  getEslintOptions,
-  clone,
-  // We remove these keys since they are needed when running the lint command but
-  // not the eslintrc command. The lint command uses ESLint's CLIEngine, but the
-  // ESLint RC format does not match the CLIEngine format exactly. We must remove
-  // anything we add that does not comply with ESLint's schemas.
-  // https://github.com/eslint/eslint/blob/9d1df92628dd4dd1e70fbb19454008e146387435/conf/config-schema.js
-  // https://github.com/eslint/eslint/blob/9d1df92628dd4dd1e70fbb19454008e146387435/lib/config/config-validator.js#L167
-  omit([
-    'failOnError',
-    'emitWarning',
-    'emitError',
-    'cwd',
-    'useEslintrc',
-    'fix',
-    'extensions',
-    'formatter'
-  ]),
-  renameKeys({ envs: 'env', baseConfig: 'extends' }),
-  evolve({
-    extends: prop('extends'),
-    globals: reduceToTrueKeys,
-    env: reduceToTrueKeys
-  })
-);
 
 const merge = (source, destination) => {
-  const sourceRules = pathOr({}, ['eslint', 'rules'], source);
-  const destinationRules = pathOr({}, ['eslint', 'rules'], destination);
+  const sourceRules = (source && source.eslint && source.eslint.rules) || {};
+  const destinationRules = (
+    destination &&
+    destination.eslint &&
+    destination.eslint.rules
+  ) || {};
   const rules = deepmerge(sourceRules, destinationRules, {
     arrayMerge(source, destination) {
       return destination;
```

Comment:
I can't select the line, since it's further down below - but there's a leftover `neutrino.options.command !== 'start'`.

I wonder if `command` could be renamed to something else so third-party middleware / end-users' `.neutrinorc.js` conditionals fail hard rather than silently not matching?